database.cpp

```diff
@@ -1963,7 +1963,10 @@ static int sqliteLoadAllSensorsCallback(void *user, int ncols, char **colval , c
             item->setValue(100);
         }
 
-        if (stateCol >= 0 && !isClip)
+        // @manup: What was the reason not to restore state for CLIP sensors?

```

Comment:
Not sure, but I figure that this was my lazy approach to always *initialize* these to defined values on startup.